packages/services/api/src/modules/audit-logs/resolvers/Query/auditLogs.ts

```diff
@@ -0,0 +1,29 @@
+import { GraphQLError } from 'graphql';
+import { AuthManager } from '../../../auth/providers/auth-manager';
+import { AuditLogManager } from '../../providers/audit-logs-manager';
+import type { QueryResolvers } from './../../../../__generated__/types.next';
+
+export const auditLogs: NonNullable<QueryResolvers['auditLogs']> = async (_parent, args, ctx) => {
+  const isAdmin = ctx.injector
```

Comment:
isAdmin refers to the Hive instance admin, not the org admin. 
please use:

    await this.authManager.ensureOrganizationOwnership({
      organization: selector.organization,
    });